package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "cls-mongoose-shim",
-  "version": "0.0.8",
+  "version": "1.0.0",
```

Comment:
bump to 1.0.0, it's been very stable using mongoose <6; so no reason to believe it won't have hit stable state for `>=6 <7`